frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -566,6 +573,13 @@ const QueueItem = ({
     cardColorDisplay()
   );
 
+  const context = {
```

Comment:
**suggestion (non-blocking):** I realize this is only 13 lines away from where it's used, but pls consider renaming to `timerContext`

Reply:
65954bd 👏